component-playground/lib/parse-component-path.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function(componentPath) {
+  // './component-group/my-component.jsx' => ('component-group/my-component')
+  // './my-component.js' => ('my-component')
+  return componentPath.match(/^\.\/?(.+)\.jsx?$/);
```

Comment:
Why the first `?`? The path will always start with `./` so the regexp can be simpler: `/^\.\/(.+)\.jsx?$/` – Please correct me if I'm wrong.